docs/sops/serverless-controller-knative-up.md

```diff
@@ -0,0 +1,93 @@
+# OpenShift Serverless Knative Serving Control Plane Readiness
+
+## Severity: Critical
+
+## Impact
+
+Any users or services using Knative Serving will be unable to get their Knative Services reconciled.
+This means, any new Knative Services might not be created, and any existing Knative Services might not be updated. Also, it might also mean that the user workload is not being served.
+
+## Summary
+
+OpenShift Serverless does not create any alerts out-of-the box. However, users can create alerts based on the metrics exposed.
+
+The metric `knative_up{namespace="openshift-serverless", type="serving_status"}` should always have a value of `1`. 
+
+This failure should only occur in extremely catastrophic scenarios as all components are deployed in HA configuration.
+
+Possible causes can be:
+- Network issues in the cluster
+- Resource starvation on the cluster
+- OpenShift Serverless upgrade issues
+
+You should check both OpenShift Serverless operator and Knative Serving control plane status.
+
+## Steps
+
+1. Check to see if all the pods are running in `openshift-serverless` namespace:
+
+```bash
+$ oc -n openshift-serverless get pods
+```
+
+2. If they are not running, look at the pod's logs/events to see what may be causing the issues. Please make sure to grab the logs/events so they can be shared with the engineering team later:
```

Comment:
I'm not sure aware of the target audience for a SOP, but I'd assume that anyone operating the cluster already knows how to do to this? Would it be sufficient to just list which pods to take a look at, similar to 
https://knative.dev/docs/install/troubleshooting/

Reply:
I like that idea from upstream, instead of doing too much of an introduction, which can be assumed as known (I guess)

Reply:
I've followed the SOPs in SRE repository in managed tenants. I would prefer keeping this format. It is them who is going to use the SOPs.